src/main/scala/devices/debug/Debug.scala

```diff
@@ -1517,10 +1517,10 @@ class TLDebugModuleInner(device: Device, getNComponents: () => Int, beatBytes: I
     import CtrlState._
 
     // This is not an initialization!
-    val ctrlStateReg = Reg(CtrlState(Waiting))
+    val ctrlStateReg = RegInit(CtrlState(Waiting))
```

Comment:
if we are turning this into a `RegInit` shouldn't we get rid of the comment above? This seems to be doing something more than the conversion to chisel3?

Reply:
I couldn't figure out how to get the type needed for Reg( ).  I have now and changed it back to the original functionality, i.e. no initialization.

Reply:
ok sounds good